app/views/films/index.html.erb

```diff
@@ -17,163 +17,257 @@
 
 <body>
 
-    <section>
-
```

Comment:
remove extra spaces to many extra spaces

if you need to add a line / space add one line simple nore more then that
like this 
```
<body>

     <section>
```
NOT like this 
```
<body>



     <section>
```